assets/src/styles/blocks/core-overrides/BlockSpacing.scss

```diff
@@ -21,17 +21,10 @@
 .wp-block-file,
 .wp-block-button,
 .page-template > h2,
-.page-template > ul,
+.page-template > ul:not(.skip-links),
```

Comment:
Be aware that adding a `:not()` selector increases the specificity of this rule, which can result in other rules in chlid themes not being applied anymore.

In general `:not()` is something to be used sparingly at it makes the code a lot harder to understand. So instead of changing the rule that applies to all ul's, can we fix it instead in the `.skip-links` class?

Reply:
Looks like putting `display: block !important;` in `.skip-links` also fixes it.

Reply:
We already have a place for `.skip-links` in [the styleguide](https://github.com/greenpeace/planet4-styleguide/blob/master/src/components/_skip-links.scss). Maybe it's a better place to override attributes.

Reply:
Won't be needed after all, we found the issue. `.page-template` is not only on the `div` wrapping the post content, it is also on the `body` element of those pages. So `.skip-links` is a direct descendant of the body it also matched.

Reply:
So we can fix the issue by changing it to `div.page-template`